.misc/deploy.pypi.sh

```diff
@@ -4,18 +4,22 @@
 set -e
 
 # Get environment variables, readily decrypted by rultor
-source ../rultor_secrets.sh
+#source ../rultor_secrets.sh
 
 # Ship it!
 echo "Uploading coala to pypi"
 pip3 install twine wheel
 python3 setup.py sdist bdist_wheel
 # Upload one by one to avoid timeout
-# Don't block merging because pypi is in error 500 mood again
-set +e
-twine upload dist/*.whl -u "$PYPIUSER" -p "$PYPIPW"
-twine upload dist/*.tar.gz -u "$PYPIUSER" -p "$PYPIPW"
-set -e
+twine upload dist/* -u "$PYPIUSER" -p "$PYPIPW" 2>&1 | tee twine_output.txt
```

Comment:
I not sure why storing `twine_output.txt` in a variable would make any difference, I can do that if necessary, is it because the name is too long?
Also, I think he suggested using a file cause it's easy to grep for a string in it.